api/apps/api/test/geo-features.e2e-spec.ts

```diff
@@ -78,6 +79,9 @@ describe('GeoFeaturesModule (e2e)', () => {
         expect(response.body.data[0].attributes.featureClassName).toEqual(
           geoFeaturesFilters.cheeta.featureClassName,
         );
+        expect(response.body.data[0].attributes.isCustom).toEqual(
+          !!response.body.data[0].attributes.projectId,
+        );
```

Comment:
I don't fully like the way we are doing kind of `white box testing` in e2e/integration specs 🤔 

may be we can extend `api/apps/api/test/upload-feature/upload-feature.e2e-spec.ts` (or reuse its fixtures?) instead relying on magic seeds?

(could be minor, but our future `us` would demand its more important 😅 )

Reply:
I don't think we need a dedicated test suite for this: we are formalizing an assumption that features with no project associated are global, while features associated to a project are custom - so, we can take advantage of the existing tests to assert exactly these assumptions, I don't see the need for further tests or seeds :)

In the end, I ended up changing things a bit, but in essence what I'm doing is:
* testing that all features in the `api/apps/api/test/geo-features.e2e-spec.ts` suite have `isCustom` set to `false`;
* testing that features uploaded as part of the `api/apps/api/test/upload-feature/upload-feature.e2e-spec.ts` suite have `isCustom` set to `true`;

Let me know if you have any concerns with this approach, or if you prefer me to test this in a different way :)

Reply:
@ikas 
> I don't think we need a dedicated test suite for this
all good, I mean exactly those you mentioned

>  I don't see the need for further tests or seeds :)
My issue with with using seeds that come from .sql whiles (`make seeds-db` etc) so tests are vulnerable to changing those.


👍